shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
Looking at what you've said, here's my opinion on it: CardA -- Front, top. CardB -- Front, bottom. CardC -- Back, entire

CardA should have `names = [CardB, CardC]` where the _last_ card in the array is the transformed (just in case in the future wizards makes a 3 part meld...) and all other entries are the other pieces needed to be connected with

CardB should have `names = [CardA, CardC]` (as reasoned above)

CardC should have `names = [CardA, CardB]` Where the ordering of the names array is the pieces used to make the card, top down (so since A was the "top" piece, it goes first. Allows us to account for future pieces added)

What do you think of this idea? I see you want to be able to distinguish between the top/bottom half and I feel this is a fair compromise on the situation.

To attempt and resolve the problems:
Number: We shall consider the numbering the same as the front side, appended with a symbol of some kind. If the front top half is "15" and the front bottom half is "28" then we shall renumber them to 15a/28a for the front and 15b/28b for the back top and bottom, respectively. This helps to resolve the issue of side labels and tracking which half of the card belongs to what.